src/main.rs

```diff
@@ -50,27 +54,23 @@ fn draw(app: &mut App, gfx: &mut Graphics, state: &mut State) {
     let width = W / 2.0;
     let scale = width / state.textures[0].width();
 
+    draw.set_blend_mode(Some(BlendMode::OVER));
+
     // Draw normal PNG
-    {
-        draw.image(&state.textures[0])
-            .translate(0.0, 0.0)
-            .scale(scale, scale);
-    }
+    draw.image(&state.textures[0])
+        .translate(0.0, 0.0)
+        .scale(scale, scale);
 
     // Draw transparent PNG
-    {
-        draw.image(&state.textures[1])
-            .translate(width, 0.0)
-            .scale(scale, scale);
-    }
+    draw.image(&state.textures[1])
+        .translate(width, 0.0)
+        .scale(scale, scale);
 
     // Draw transparent PNG with RenderTexture
     {
-        let mut d = gfx.create_draw();
-        d.set_size(TEX_W, TEX_H);
+        let mut d = state.rt1.create_draw();
         d.clear(Color::TRANSPARENT);
-        d.image(&state.textures[1])
-            .scale_from((TEX_W / 2.0, TEX_H / 2.0), (1.0, -1.0));
```

Comment:
v0.8 takes care already of flipping the y-axis for Rendering to `RenderTexture` when using the `Draw2D` API.